File: Python/Program/Component.py
class Component:
    def __init__(self, brand="", model=""):
        self.brand = brand
        self.model = model

    def set_brand(self, brand):
        self.brand = brand

    def set_model(self, model):
        self.model = model

    def get_brand(self):
        return self.brand

    def get_model(self):
        return self.model

File: Java/Program/Component.java
public class Component {
    protected String brand;
    protected String model;

    public Component() {}

    public Component(String brand, String model) {
        this.brand = brand;
        this.model = model;
    }

    public void setBrand(String brand) { this.brand = brand; }
    public void setModel(String model) { this.model = model; }

    public String getBrand() { return brand; }
    public String getModel() { return model; }
}
